doxia-core/pom.xml

```diff
@@ -53,6 +53,11 @@ under the License.
       <artifactId>commons-text</artifactId>
       <version>1.10.0</version>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.11.0</version>
+    </dependency>
```

Comment:
Wrong scope here

Reply:
Probably still a few to go, but with the additional files I've added  this is now needed in compile scope.